packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -0,0 +1,159 @@
+import { JsonRpcProvider, Provider } from "@ethersproject/providers"
+import { Chains } from "@streamr/config"
+import { Wallet } from "@ethersproject/wallet"
+import { parseEther } from "@ethersproject/units"
+import { Logger, wait, waitForCondition } from '@streamr/utils'
+import fetch from "node-fetch"
+
+import type { IERC677, Operator } from "@streamr/network-contracts"
+import type { StreamRegistry } from "@streamr/network-contracts"
+
+import { tokenABI } from "@streamr/network-contracts"
+import { streamRegistryABI } from "@streamr/network-contracts"
+import { Contract } from "@ethersproject/contracts"
+
+import { deployOperatorContract } from "./deployOperatorContract"
+import { deploySponsorship } from "./deploySponsorshipContract"
+import { MaintainOperatorValueService } from "../../../../src/plugins/operator/MaintainOperatorValueService"
+import { OperatorServiceConfig } from "../../../../src/plugins/operator/OperatorPlugin"
+
+const config = Chains.load()["dev1"]
+const adminPrivKey = "0x4059de411f15511a85ce332e7a428f36492ab4e87c7830099dadbf130f1896ae"
```

Comment:
This could refer to `ADMIN_WALLET_PK` constant if we export that from `smartContractUtils.ts`.